reader.go

```diff
@@ -460,6 +474,19 @@ func (r *Reader) SeekToRow(rowIndex int64) error {
 	return nil
 }
 
+// GetKeyValueMetadata returns the value of the key/value metadata associated
+// with the given key.
+// The method returns an empty string if the reader has not been configured with
+// a file that has key/value metadata.
+func (r *Reader) GetKeyValueMetadata(key string) string {
```

Comment:
For consistency, would it make sense to name this `Lookup` like we do for https://pkg.go.dev/github.com/parquet-go/parquet-go#File.Lookup ?

I don't have a strong feeling about this, just wanted to get your thoughts on it.

Reply:
Alternatively, we could expose the underlying `File` with a method like this:
```go
func (r *Reader) File() *File { return r.file }
```
Then we don't have to duplicate access to the metadata or other aspects of it.

Reply:
I think renaming to Lookup would be better here! 
`r.file` is a `reader` type here, so we would need to add a field to expose it. Also, it might not be great to expose it this way as it could lead to some convoluted usage. Would be better to use `OpenFile` directly. :)